gsa/src/gmp/models/certbund.js

```diff
@@ -29,26 +29,26 @@ class CertBundAdv extends Info {
   parseProperties(elem) {
     const ret = super.parseProperties(elem, 'cert_bund_adv');
 
-    ret.severity = parseSeverity(ret.max_cvss);
+    ret.severity = parseSeverity(elem.max_cvss);
     delete ret.max_cvss;
 
     ret.categories = [];
     ret.description = [];
     ret.cves = [];
-    ret.additional_information = [];
+    ret.additionalInformation = [];
 
-    if (isDefined(ret.raw_data) && isDefined(ret.raw_data.Advisory)) {
+    if (isDefined(elem.raw_data) && isDefined(elem.raw_data.Advisory)) {
       const {raw_data} = ret;
```

Comment:
Better to use elem instead of ret here too
```suggestion
       const {raw_data} = elem;
```